README.rst

```diff
@@ -0,0 +1,63 @@
+# reana-auth-krb5
+
+Required information and inputs:
+
+
+
+## Building the image
+
+No build arguments needed.
+
+
+===========================
+REANA Environment Kerberos5
+===========================
+
+About
+=====
+
+``reana-auth-krb5`` provides a container image with encapsulated runtime
+execution environment for creating an renewing Kerberos tokens. The container image
+includes no additional logic or libraries, just the bare minimum to support the Kerberos operations.
+
+``reana-auth-krb5`` was developed for use in the `REANA <http://reana.readthedocs.io/>`_ reusable research data analysis platform.
+
+Usage
+=====
+
+You can use ``reana-auth-krb5`` as a base image, however it was built
+to be used as a sidecar container with the single purpose of refreshing the Kerberos token for long running jobs.
+Once obtained, the token is shared with the main container through the token cache directory (``KRB5CCNAME``).
+
+Some concrete usage examples will be provided later on.
+
+Development
+===========
+
+Running the container and successfully obtaining a shared token on a sidecar container requires additional information and inputs:
+
+- The [Kerberos cache](https://web.mit.edu/kerberos/krb5-1.12/doc/basic/ccache_def.html) location to be shared, configured through the ``KRB5CCNAME`` environment variable
+- [Kerberos configuration](https://web.mit.edu/kerberos/krb5-1.12/doc/admin/conf_files/krb5_conf.html) at ``/etc/krb5.conf`` (overridable)
+
+If you want to try it locally, a Kerberos token can be obtained via:
+
+```shell
+$ docker run -i -t --rm reanahub/reana-auth-krb5 /bin/bash
+> kinit -kt /path/to/keytab_file username@server.org
+> klist
+```
+
+More information
+================
+
+For more information about `REANA <http://reanahub.io/>`_ reusable research data
```

Comment:
`www.reana.io` or `www.reanahub.io` -- the domain without leading `www` is not load-balanced without CNAME so it may be more downtimes when compared to `www`.